controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -194,159 +185,164 @@ void dynamic_channel_selection_task::handle_event(int event_type, void *obj)
         TASK_LOG(ERROR) << "obj == nullptr";
         return;
     }
-    auto event_radio_mac = network_utils::mac_to_string(((sMacAddr *)obj)->oct);
-    if (m_radio_mac == event_radio_mac) {
-        bool handle_event = false;
-        switch (eEvent(event_type)) {
-        case eEvent::TRIGGER_SINGLE_SCAN: {
-            TASK_LOG(DEBUG) << "TRIGGER_SINGLE_SCAN received";
-            auto trigger_single_scan_event = reinterpret_cast<sTriggerSingleScan_event *>(obj);
-            handle_event                   = true;
-            TASK_LOG(DEBUG) << "TRIGGER_SINGLE_SCAN handled on:"
-                            << network_utils::mac_to_string(
-                                   trigger_single_scan_event->radio_mac.oct);
-            m_is_single_scan_pending = true;
-            break;
-        }
-        case eEvent::SCAN_TRIGGERED: {
-            TASK_LOG(DEBUG) << "SCAN_TRIGGERED received";
-            if (FSM_IS_IN_STATE(WAIT_FOR_SCAN_TRIGGERED)) {
-                auto scan_triggered_event = reinterpret_cast<sScanTriggered_event *>(obj);
-                handle_event              = true;
-                TASK_LOG(DEBUG) << "SCAN_TRIGGERED handled on:"
-                                << network_utils::mac_to_string(
-                                       scan_triggered_event->radio_mac.oct);
-
-                set_events_timeout(SCAN_RESULTS_DUMP_WAIT_TIME_MSEC);
-                dcs_wait_for_event(eEvent::SCAN_RESULTS_DUMP);
-
-                FSM_MOVE_STATE(WAIT_FOR_RESULTS_READY);
-            } else {
-                TASK_LOG(DEBUG) << "ignoring SCAN_TRIGERRED event,"
-                                << " not in state WAIT_FOR_SCAN_TRIGGERED";
-            }
-            break;
+
+    auto event_radio_mac = beerocks::net::network_utils::mac_to_string(((sMacAddr *)obj)->oct);
+    if (m_radio_mac != event_radio_mac) {
+        TASK_LOG(ERROR) << "event : " << s_ar_events[int(eEvent(event_type))]
+                        << " in state: " << s_ar_states[int(m_fsm_state)]
+                        << " for mac: " << event_radio_mac;
```

Comment:
also here - `event_radio_mac` if it will be a simple `sMacAddr` - has the `<<` operator for `cout` so you can use it in prints.